### PR TITLE
CLI: fix defaults; make consistent with admin cli

### PR DESCRIPTION
**Pull request by kirg**

### CODE REVIEW

File: cmd/tools/admin/main.go [hunk 1]
```diff
@@ -22,10 +22,22 @@ package main
 
 import (
 	"os"
+	"time"
 
 	"github.com/codegangsta/cli"
 	"github.com/uber/cherami-server/common"
 	"github.com/uber/cherami-server/tools/admin"
+	com "github.com/uber/cherami-server/tools/common"
```
Comment (datoug) [inline cmd/tools/admin/main.go]: rename to toolscommon for better readability

**Comment by datoug** [comment]
We probably should just deprecate these commands, which are duplicates from cherami-cli. I don't see the value of having the same commands in two tools, and they're causing code duplicates like this.

Or as @thuningxu has mentioned before, we can fold these two tools into a single binary, and only expose the 'admin' functionalities to admins. Although I personally slightly prefer the current solution.

File: cmd/tools/cli/main.go [hunk 2]
```diff
@@ -123,7 +122,7 @@ func main() {
 						},
 						cli.StringSliceFlag{
 							Name:  "zone_config, zc",
-							Usage: "Zone configs for multi_zone destinations. Format for each zone should be \"ZoneName,AllowPublish,AllowConsume,ReplicaCount\". For example: \"zone1,true,true,3\"",
+							Usage: "Zone configs for multi_zone destinations. Format for each zone should be \"Zone,AllowPublish,AllowConsume,ReplicaCount\". Ex: \"sjc1a,true,true,3\"",
```
Comment (datoug) [inline cmd/tools/cli/main.go]: we don't want to use 'sjc1' in the open source repo

File: cmd/tools/cli/main.go [hunk 3]
```diff
@@ -27,18 +27,17 @@ import (
...
-
-	strDelaySeconds = `Delay to introduce to all messages, in seconds.`
-	intDelaySeconds = 0 // zero delay, by default
+	usageCGStartTime                  = `Consume messages newer than this time in unix-nanos (default: Now; ie, consume no previously published messages)`
+	usageCGLockTimeoutSeconds         = `Acknowledgement timeout for prefetched/received messages`
+	usageCGMaxDeliveryCount           = `Max number of times a message is delivered before it is sent to the DLQ (dead-letter queue)`
+	usageCGSkipOlderMessagesInSeconds = `Skip messages older than this duration, in seconds ('0' to skip none)`
+	usageCGDelaySeconds               = `Delay, in seconds, to defer all messages by`
+	usageCGOwnerEmail                 = "Owner email"
+	usageCGZoneConfig                 = "Zone configs for multi-zone CG. For each zone, specify \"Zone,PreferedActiveZone\"; ex: \"dca1a,false\""
```
Comment (datoug) [inline cmd/tools/cli/main.go]: no 'dca1' or 'sjc1' in the open source repo

File: cmd/tools/admin/main.go [hunk 4]
```diff
@@ -22,10 +22,22 @@ package main
...
+)
+
+const (
+	usageCGStartTime                  = `Consume messages newer than this time in unix-nanos (default: Now; ie, consume no previously published messages)`
+	usageCGLockTimeoutSeconds         = `Acknowledgement timeout for prefetched/received messages`
+	usageCGMaxDeliveryCount           = `Max number of times a message is delivered before it is sent to the DLQ (dead-letter queue)`
+	usageCGSkipOlderMessagesInSeconds = `Skip messages older than this duration, in seconds ('0' to skip none)`
+	usageCGDelaySeconds               = `Delay, in seconds, to defer all messages by`
+	usageCGOwnerEmail                 = "Owner email"
+	usageCGZoneConfig                 = "Zone configs for multi-zone CG. For each zone, specify \"Zone,PreferedActiveZone\"; ex: \"dca1a,false\""
```
Comment (datoug) [inline cmd/tools/admin/main.go]: no dca1

File: tools/common/lib.go [hunk 5]
```diff
@@ -71,6 +71,10 @@ const (
 	DestinationType = "DST"
 	// ConsumerGroupType is the name for entity type for consumer group in listEntityOps
 	ConsumerGroupType = "CG"
+	// DefaultUnconsumedMessagesRetention is the default value for unconsumed messages retention
+	DefaultUnconsumedMessagesRetention = 7200
+	// DefaultConsumedMessagesRetention is the default value for consumed messages retention
+	DefaultConsumedMessagesRetention = 3600
```
Comment (datoug) [inline tools/common/lib.go]: these values are too low IMO. I suggest to change to 1d and 3d

**Comment by kirg** [comment]
About admin vs cli . I think Kobe is working on something to that effect. So not dealing with it in this diff.

**Comment by kobeyang** [comment]
Yes, I'm working on refactoring the cherami-cli and cherami-admin code. Will send out a PR later.

File: cmd/tools/admin/main.go (hunk 4 above)
Comment (datoug) [inline cmd/tools/admin/main.go]: still has dca1 here

**Pull request merged**